glean-core/python/tests/test_glean.py

```diff
@@ -0,0 +1,13 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+import glean
+
+
+def test_smoke_test(tmpdir):
+    """
+    A very simple smoke test.
+    """
+    glean.Glean.initialize(glean.Configuration(), tmpdir)
```

Comment:
Instead of relying on the exception, if we keep the same behaviour as Kotlin, let's add an assertion for checking if we're initialized.

Reply:
Sure thing.